compiler/src/iree/compiler/Codegen/Common/Transforms.h

```diff
@@ -19,5 +20,9 @@ void populateTileAndDistributeToWorkgroupsPatterns(
 /// hal.interface.binding.subspan.
 void populateReshapeToInterfaceTensorPatterns(RewritePatternSet &patterns);
 
+/// Sets compilation configuration annotated in the incoming IR.
```

Comment:
Could we move this to `Utils`?

Reply:
sure

Reply:
Regarding moving this to Utils, this function requires IREE::Codegen::CompilationInfoAttr which is part of the IREECodegenDialect but that library requires the Utils lib so we end up with a circular dependency. So I don't think that will work. Should I keep it in Common or do you have any suggestions on how to get around the circular dependency?